website/docs/downloads/development-builds.md

```diff
@@ -92,8 +92,8 @@ function set_ci_status(workflow_file, os_name, description, page = 1) {
 document.addEventListener("DOMContentLoaded", () => {
     set_ci_status("linux.yml", "linux", "Linux");
     set_ci_status("macos.yml", "macos", "macOS");
-    set_ci_status("windows-msys2.yml", "msys2", "Windows MSYS2 builds");
-    set_ci_status("windows-msvc.yml", "windows", "Windows MSVC builds");
+    set_ci_status("windows-msys2.yml", "msys2", "Windows MSYS2 builds (64-bit ZIP, 64-bit Installer with MSYS2 and MSVC)");
+    set_ci_status("windows-msvc.yml", "windows", "Windows MSVC builds (32-bit ZIP, 64-bit ZIP)");
```

Comment:
I don't like how this info looks inlined in the table. Please break them out below the table as footnotes. You might want to look up how to do proper footnotes with MkDocs first.

<img width="804" alt="image" src="https://user-images.githubusercontent.com/698770/217956551-464a2d84-a601-4b2f-803c-a4d864f5346a.png">

Reply:
Agree. I also wondered how it will look... and now that I see it: don't like it :)